crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -14,7 +14,7 @@ use std::sync::Arc;
 use subspace_networking::libp2p::multiaddr::Protocol;
 use subspace_networking::{
     peer_id, BootstrappedNetworkingParameters, Config, MemoryProviderStorage,
-    ParityDbProviderStorage,
+    ParityDbProviderStorage, SWARM_PENDING_TO_ESTABLISHED_CONNECTIONS_FACTOR,
```

Comment:
Do we need to expose this constant though? Established connections are constants in the binary only, it seems inconsistent to expose such things from the library.